### PR TITLE
New studies added

### PR DESCRIPTION
**Pull request by stephpenn1**
Pretty straightforward but I'm wondering if i could simplify lines `24-26` cause its repeated in `29-31`

### CODE REVIEW

File: rs-synthesis.Rmd [hunk 1]
```diff
@@ -14,13 +14,23 @@ library(forcats)
 library(DT)
 library(metafor)
 
+theme_set(theme_bw())
```
Comment (bpbond) [inline rs-synthesis.Rmd:17]: Wait, what?!? Are you giving up on `theme_minimal()`?

**Review by bpbond — APPROVED** [review]
I don't see a repetition problem myself. What does the treatment effect plot look like with these data included?

File: rs-synthesis.Rmd (hunk 1 above)
Comment (stephpenn1) [reply rs-synthesis.Rmd:17]: Ugh, maybe I should stay true to myself. Or compromise with `theme_light()`

**Comment by stephpenn1** [comment]
This one?

image

File: rs-synthesis.Rmd [hunk 2]
```diff
@@ -14,7 +14,7 @@ library(forcats)
 library(DT)
 library(metafor)
 
-theme_set(theme_bw())
+theme_set(theme_minimal())
```
Comment (bpbond) [inline rs-synthesis.Rmd:17]: 😆 👍

**Comment by bpbond** [comment]
Looks _beautiful_. Thank you!

**Comment by stephpenn1** [comment]
Would like to change the color scale range to easily identify values but for later

**Pull request merged**